locales/en-US/experiments.ftl

```diff
@@ -29,6 +29,22 @@ cliqzToursteps0Copy = Start typing in the URL bar to see real-time results.
 cliqzToursteps1Copy = Open a new tab and add your favorite websites for one-click access.
 cliqzToursteps2Copy = You can always give us feedback or disable Cliqz from Test Pilot.
 cliqzEolwarning = This experiment is ending soon. If you have it enabled, you can keep it for now, and we will help you migrate to the next version when it becomes available.
+# Please no not Localize the name Firefox Color
```

Comment:
Note, these comments will get blown away when this file gets recompiled...i filed #3563 as a long term fix